src/wallet/signer.ts

```diff
@@ -13,6 +13,11 @@ export interface Signer {
    */
   getPublicKey(): Promise<Uint8Array>;
 
+  /**
+   * Returns the signer's Secp256k1-compressed private key
```

Comment:
I would note that this is not the compressed private key, but the actual raw private key

Reply:
@zivkovicmilos
I missed something in the comments
I changed it to `actual raw private key`.